codemap/api-hot.md

```
@@ -175,6 +175,16 @@ codemap/signatures/ · type-directed search: ./dev api "<type>"
 - `waitCatch :: Show err => AsyncTask err result -> Task err2 (Result Text result)`  <!-- 2 call sites -->
 - `race :: Show err => Task err a -> Task err b -> Task err (RaceWinner a b)`  <!-- 1 call sites -->
 
+## Uuid
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)